### PR TITLE
Update a couple of python dependencies

### PR DESCRIPTION
**Pull request by TobyRoseman**
Update dependencies (TensorFlow and Pillow). Must still rely on the old version TensorFlow for Python 2.7; it is the last version to support 2.7.

### CODE REVIEW

**Comment by TobyRoseman** [comment]
Internal test pass

File: scripts/requirements.txt [hunk 1]
```diff
@@ -7,7 +7,8 @@ scipy==1.2.1
 resampy==0.2.1
 scikit-learn==0.20.3
 statsmodels==0.9.0
-tensorflow==2.0.0
+tensorflow==2.0.1; python_version > '2.7'
+tensorflow==2.0.0; python_version < '3.5'   # Last version to support Python 2.7
```
Comment (nickjong) [inline scripts/requirements.txt:11]: Did they really drop 2.7 support between TF2.0.0 and TF2.0.1?

Comment (nickjong) [reply scripts/requirements.txt:11]: Hmm, this page on PyPI suggests that TF2.0.1 supports Python 2.7? https://pypi.org/project/tensorflow/2.0.1/

Comment (TobyRoseman) [reply scripts/requirements.txt:11]: Sadly, TensorFlow did drop support for Python 2.7 between 2.0.0 and 2.0.1. I've tried and you can't install 2.0.1 in 2.7.

Yes, Python 2.7 is listed at the bottom on the left hand side on the PyPI page for 2.0.1. However if you click on [Download Files](https://pypi.org/project/tensorflow/2.0.1/#files), there are no wheel for 2.7.

Comment (nickjong) [reply scripts/requirements.txt:11]: Wow

**Pull request merged**